back/db/structure.sql

```diff
@@ -117,6 +117,7 @@ ALTER TABLE IF EXISTS ONLY public.custom_field_option_images DROP CONSTRAINT IF
 ALTER TABLE IF EXISTS ONLY public.nav_bar_items DROP CONSTRAINT IF EXISTS fk_rails_34143a680f;
 ALTER TABLE IF EXISTS ONLY public.volunteering_volunteers DROP CONSTRAINT IF EXISTS fk_rails_33a154a9ba;
 ALTER TABLE IF EXISTS ONLY public.phase_files DROP CONSTRAINT IF EXISTS fk_rails_33852a9a71;
+ALTER TABLE IF EXISTS ONLY public.embeddings_similarities DROP CONSTRAINT IF EXISTS fk_rails_2e90c705c4;
```

Comment:
The structure.sql file was manipulated manually and should be regenerated.

In the end, the table was created manually with:
```
CREATE TABLE change_nhs_uk.embeddings_similarities (
    id uuid DEFAULT gen_random_uuid() NOT NULL,
    embedding shared_extensions.vector(1024),
    idea_id uuid NOT NULL,
    created_at timestamp(6) without time zone NOT NULL,
    updated_at timestamp(6) without time zone NOT NULL
);
```